pattern-library/css/edx-pattern-library-ltr.css

```diff
@@ -64,6 +64,11 @@
     content: "";
     display: table; }
 
+.list:last-child, .list-bulleted:last-child, .list-ordered:last-child, .list-inline:last-child, .list-links:last-child, .list-grouped:last-child, .list .item:last-child, .list-bulleted .item:last-child, .list-ordered .item:last-child, .list-inline .item:last-child, .list-links .item:last-child, .list-grouped .item:last-child, .list .item-complex:last-child {
+  margin-bottom: 0;
+  padding-bottom: 0;
+  border-bottom: none; }
```

Comment:
This is the compiled CSS file (used locally). Do you want me to try to change the output of this file/compilation?